package.json

```diff
@@ -59,20 +59,21 @@
   "devDependencies": {
     "@okta/okta-auth-js": "~2.6.0",
     "@sindresorhus/to-milliseconds": "^1.0.0",
-    "autoprefixer": "6.7.7",
-    "axe-core": "2.0.7",
-    "axe-webdriverjs": "0.4.0",
-    "axios": "^0.18.0",
+    "autoprefixer": "^9.6.1",
+    "axe-core": "^3.3.1",
+    "axe-webdriverjs": "^2.2.0",
+    "axios": "^0.19.0",
     "babel-core": "6",
     "babel-loader": "7",
-    "babel-plugin-istanbul": "^5.0.1",
+    "babel-plugin-istanbul": "^5.2.0",
     "babel-plugin-transform-runtime": "^6.23.0",
     "babel-preset-env": "^1.6.0",
     "backbone": "1.2.1",
     "cssnano": "^4.1.7",
     "dotenv": "^8.0.0",
+    "dyson": "^2.0.5",
     "eslint": "^5.9.0",
-    "fs-extra": "^7.0.1",
+    "fs-extra": "^8.1.0",
```

Comment:
Is it expected that all these other packages are also getting updated?

Reply:
it's just a bonus work I did and it's independent to the mock server change.